### ISSUE TITLE
Incorrect "Incorrect list-item indent: add 2 spaces (remark-lint:list-item-indent)"

### ISSUE THREAD

**Issue by jpluimers** [remarkjs/remark-lint#178]
In Atom.io, the below markdown incorrectly throws "Incorrect list-item indent: add 2 spaces (remark-lint:list-item-indent)" warnings.

How can I track this down on a plain console/terminal with a small demo file?

``` markdown
# Heading 1

## List at left margin

- one
- two
  - two.one

## List indented two

  - one
  - two
    - two.one

## results

> Warning	remark-lint	Incorrect list-item indent: add 2 spaces (remark-lint:list-item-indent)	5:3
> Warning	remark-lint	Incorrect list-item indent: add 2 spaces (remark-lint:list-item-indent)	6:3
> Warning	remark-lint	Incorrect list-item indent: add 2 spaces (remark-lint:list-item-indent)	7:5
> Warning	remark-lint	Incorrect indentation before bullet: remove 2 spaces (remark-lint:list-item-bullet-indent)	11:3
> Warning	remark-lint	Incorrect indentation before bullet: remove 2 spaces (remark-lint:list-item-bullet-indent)	12:3
> Warning	remark-lint	Incorrect list-item indent: add 2 spaces (remark-lint:list-item-indent)	13:7
```

**Comment by wooorm** [comment]
Hi Jeroen, you can configure rules, see the [examples](https://github.com/remarkjs/remark-lint/blob/master/packages/remark-lint-list-item-indent/readme.md) for this rule on how to configure your preferred style!

**Comment by wooorm** [comment]
tab-size is safer across markdown flavours: it makes sure that nested indented code blocks work the same everywhere!

If you’re in atom, you should be able to configure it. Or use `linter-remark` directly!

**Comment by imranbaloch** [comment]
Hi woorom, I am confused why when if I have(2 space or 4 space),

-  Install [ABC](https://ABC.org)
or
-    Install [ABC](https://getcomposer.org)

It says `Incorrect list-item indent: add 2 spaces`. How to fix it?

**Comment by jpluimers** [comment]
Could it also check the the current settings and warn about more reasonable defaults?

**Comment by adelespinasse** [comment]
Note to anyone seeing this error: Apparently the default configuration requires 3 spaces **between** the bullet character and the text. So it doesn't seem to really be about indentation (as I understand it). Changing from this:
```
* One
* Two
* Three
```
to this:
```
*   One
*   Two
*   Three
```
fixed it for me. I had to determine this experimentally since the documentation doesn't seem to mention it.

(I don't run remark-link in my own environment, but it was enabled by default in Codacy (auto-checker for pull requests), and I thought it would be easier and better to conform to default style guidelines rather than try to figure out how to change the configuration.)

**Comment by bkdotcom** [comment]
My $0.02:  This is a dumb default
